UI/DesignWnd.cpp

```diff
@@ -210,24 +212,20 @@ namespace {
     SavedDesignsManager& GetSavedDesignsManager()
     { return SavedDesignsManager::GetSavedDesignsManager(); }
 
-    void WriteDesignToFile(int design_id, boost::filesystem::path& file) {
-        const ShipDesign* design = GetShipDesign(design_id);
-        if (!design)
-            return;
-
+    void WriteDesignToFile(const ShipDesign& design, boost::filesystem::path& file) {
```

Comment:
Seems like there should be some sent to InfoLogger when the design write succeeds, and possibly also to the messages window...